analytics.js

```diff
@@ -8646,6 +8646,7 @@ var camelSplitter = /(.)([A-Z]+)/g;
  */
 
 function uncamelize (string) {
+  if (!string) return '';
```

Comment:
This change needs to be made in the `uncamelize ` dependency, not here. The next a.js build will get rid of this change.

Reply:
@stephenmathieson can this be closed now?